benchmarks/benchmark_experiment.py

```diff
@@ -114,34 +116,32 @@ def _add_experiment_env(self, experiment_config):
         process_env["XRT_TPU_CONFIG"] = "localservice;0;localhost:51011"
       elif is_xla_device_available("CUDA"):
         process_env["GPU_NUM_DEVICES"] = "1"
-    elif not experiment_config["xla"] and is_xla_device_available(
-        experiment_config["accelerator"].upper()):
+    elif experiment_config["xla"] is None:
       # In non-xla CPU training experiments, an env var is still needed if an
       # xla device exists, or there will be "Missing XLA configuration" error.
-      process_env["PJRT_DEVICE"] = experiment_config["accelerator"].upper()
+      if is_xla_device_available(experiment_config["accelerator"].upper()):
+        process_env["PJRT_DEVICE"] = experiment_config["accelerator"].upper()
```

Comment:
Is this because, even if we aren't using XLA, we still have to populate `PJRT_DEVICE` env var? If so, wouldn't it be better just to:

```python
...
else:
    process_env["PJRT_DEVICE"] = "CPU"  # should be available everywhere
```

Reply:
The change here merely restructures the code to switch on the xla value. 

The reason for setting PJRT_DEVICE is in the comment "In non-xla CPU training experiments, an env var is still needed if an xla device exists, or there will be Missing XLA configuration error."

I am guessing it would work with setting PJRT_DEVICE to CPU always but even then I think might be cleaner to only set it when there is an XLA device. I can change it but can't really test that case